src/QSymbolicsBase/QSymbolicsBase.jl

```diff
@@ -151,12 +158,13 @@ end
 
 # TODO check that this does not cause incredibly bad runtime performance
 # use a macro to provide specializations if that is indeed the case
-propsequal(x,y) = all(n->getproperty(x,n)==getproperty(y,n), propertynames(x))
+propsequal(x,y) = all(n->isequal(getproperty(x,n),getproperty(y,n)), propertynames(x))
 
 ##
 # Most symbolic objects defined here
 ##
 
+include("objects.jl")
```

Comment:
this probably needs more descriptive name. Something like `literals` or `base_litrals` or `literal_objects`